dotCMS/src/main/java/com/dotcms/auth/providers/jwt/beans/UserToken.java

```diff
@@ -53,8 +54,8 @@ public UserToken(final String id, final String subject, final String issuer, fin
 	 *            - The expiration date of the token.
 	 */
     public UserToken(final String id, final String subject, final String issuer, final Date modificationDate,
-            long ttlMillis, final Map<String,Object> claims) {
-        this(id,subject,issuer,modificationDate,new Date(System.currentTimeMillis()+ ttlMillis), claims);
+            long ttlMillis, final Map<String,Object> claims, final String skinId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'ttlMillis' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)